docs/incidents/2019-02-06-multiple-file-uploads.md

```diff
@@ -0,0 +1,81 @@
+# 2019-02-06
+
+**Incident Leader: Peter**
+
+## Description
+
+Something went wrong and caused this problem.
+
+## Timeline
+
+All times UTC.
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads being done beforehand
+
+Peter understood that the file uploads issues were all to be parked, in favor of documentation
```

Comment:
I believe Hannah your understanding was that the documentation work was going to be done too.